Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -314,12 +411,33 @@ private void Awake()
             moveLogic = new ManipulationMoveLogic();
             rotateLogic = new TwoHandRotateLogic();
             scaleLogic = new TwoHandScaleLogic();
+
+            InitializeElastics();
         }
         private void Start()
         {
             rigidBody = HostTransform.GetComponent<Rigidbody>();
             constraints = new ConstraintManager(gameObject);
         }
+        private void Update()
+        {
+            if (!isManipulationStarted)
+            {
+                if (elasticTypes.HasFlag(TransformFlags.Move) && translationElastic != null && translationElastic.GetCurrentVelocity().magnitude > 0.001f)
```

Comment:
Will only continue simulating the spring if the user is not actively manipulating the object, if the elastic is non-null, if that particular type of elastic is enabled, and if the current elastic velocity is above the threshhold.